src/main/webapp/i18n/en/modelingExercise.json

```diff
@@ -7,6 +7,8 @@
                 "editLabel": "Edit Modeling Exercise",
                 "importLabel": "Import Modeling Exercise"
             },
+            "create": "Create Modeling",
+            "import": "Import Modeling",
```

Comment:
I think it should contain "exercise" and we should not duplicate translations keys